bool[3] Whiteline